README.md

```diff
@@ -3,7 +3,7 @@ void-elements
 
 ### Array of "void elements" defined by the HTML specification
 
-Exports an Array of "void element" node names as defined by the HTML spec.
+Exports a Map/Object of "void element" node names as defined by the HTML spec.
```

Comment:
I feel "hash" is more used but maybe it's just me coming from Perl.

But in any case drop the capitalization. It's wrong for Map and feels odd even for Object.